src/netif.rs

```diff
@@ -356,6 +356,16 @@ impl EspNetif {
         Cow::Owned(from_cstr(&netif_name).into_owned())
     }
 
+    pub fn get_mac(&self) -> Result<[u8; 6], EspError> {
```

Comment:
You need to fix formatting. Also, instead of matching on the result, please use the `?` Rust operator, which is more idiomatic.